spec/inertia/sharing_spec.rb

```diff
@@ -54,52 +54,57 @@
     it { is_expected.to eq props.merge({ errors: errors }) }
   end
 
-  context 'multithreaded intertia share' do
+  context 'when multithreaded with shared data' do
```

Comment:
Could you undo the change to this spec? The updated code should be threadsafe just by design (making this spec unncessary), but I'd prefer to test for a regression against the original spec.

Reply:
In the old implementation there was this comment:

        # Would prefer to make this a second get request, but RSpec will overwrite
        # the @response variable if another request is made in the second thread.
        # This simulates the relevant effects of another call to a different
        # controller with different values for inertia_share
        
  But i refactor to make another request and it working fine